runtime/shared_common/SharedCache.hpp

```diff
@@ -117,6 +117,10 @@ class SH_SharedCache
 	virtual J9SharedClassConfig* getSharedClassConfig() = 0;
 
 	virtual IDATA aotMethodOperation(J9VMThread* currentThread, char* methodSpecs, UDATA reason) = 0;
+	
+	virtual void* getAddressFromJ9ShrOffset(J9ShrOffSet offSet) = 0;
```

Comment:
Composite parameter type.

Reply:
Fixed

Reply:
I am thinking about keeping this one. If I change it to pointer, I need to have a local variable J9ShrOffSet localoffSet to copy the offSet in the cache pass &localoffSet to getAddressFromJ9ShrOffset().

Reply:
> I need to have a local variable J9ShrOffSet localoffSet to copy the offSet

Actually I don't need that. It is fixed now.